astroquery/vizier/core.py

```diff
@@ -325,7 +325,7 @@ def query_object_async(self, object_name, catalog=None, radius=None,
 
     def query_region_async(self, coordinates, radius=None, inner_radius=None,
                            width=None, height=None, catalog=None,
-                           get_query_payload=False, cache=True,
+                           get_query_payload=False, frame='fk5', cache=True,
```

Comment:
This has to be the last kwarg to not break API, or make all kwargs kwarg only with adding a `*` after coordinates. I would prefer the latter.